app.go

```diff
@@ -160,6 +160,10 @@ func (a *App) Setup() {
 		a.categories = a.categories.AddCommand(command.Category, command)
 	}
 	sort.Sort(a.categories)
+
+	if a.Metadata == nil {
+		a.Metadata = make(map[string]interface{})
+	}
```

Comment:
ah, just that? sorry, I misunderstood your comment. 😉 I cannot reopen the PR, but I can open a new one.